cfy_manager/status_reporter/status_reporter.py

```diff
@@ -41,11 +41,10 @@
     BROKER_STATUS_REPORTER,
     MANAGER_STATUS_REPORTER
 )
-
+VERBOSE_HELP_MSG = "Used to give more verbose output."
 SCRIPTS_PATH = join(STATUS_REPORTER_DIR, 'scripts')
 
 logger = get_logger(STATUS_REPORTER)
-setup_console_logger()
```

Comment:
the main issue is that calling setup_console_logger() here, makes any further calls (ones done inside of actual functions) be ignored due to the global "only call once" mechanic